ubuntu_requirement_installation.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+set -ex
+
+source lib/logging.sh
+
+# sudo apt install -y libselinux-utils
+# if selinuxenabled ; then
+#     sudo setenforce permissive
+#     sudo sed -i "s/=enforcing/=permissive/g" /etc/selinux/config
+# fi
+
+# Update to latest packages first
+sudo apt -y update
+
+# Install EPEL required by some packages
+# if [ ! -f /etc/yum.repos.d/epel.repo ] ; then
+#     if grep -q "Red Hat Enterprise Linux" /etc/redhat-release ; then
+#         sudo yum -y install http://mirror.centos.org/centos/7/extras/x86_64/Packages/epel-release-7-11.noarch.rpm
+#     else
+#         sudo yum -y install epel-release --enablerepo=extras
+#     fi
+# fi
+
+# Work around a conflict with a newer zeromq from epel
+# if ! grep -q zeromq /etc/yum.repos.d/epel.repo; then
+#   sudo sed -i '/enabled=1/a exclude=zeromq*' /etc/yum.repos.d/epel.repo
+# fi
+
+# Install required packages
+# python-{requests,setuptools} required for tripleo-repos install
+sudo apt -y install \
+  net-tools \
+  crudini \
+  curl \
+  dnsmasq \
+  figlet \
+  golang \
```

Comment:
this might be too old. Use the specific repo to install 1.12

Reply:
The only way to install the latest version of Goland (as far as my investigation) is to download the binary and move it to /usr/local/bin/. I will do it, but it is not a good solution in the long run.